distributed/http/worker/prometheus/core.py

```diff
@@ -112,6 +112,15 @@ def collect(self):
             value=self.server.transfer_outgoing_count,
         )
 
+        yield CounterMetricFamily(
+            self.build_name("transfer_outgoing_bytes_total"),
+            (
+                "Total size of data transferred to other workers "
+                "since the worker was started"
```

Comment:
```suggestion
                "Total size of data transfers to other workers "
                "since the worker was started (including in-progress and failed transfers)"
```